mountpoint-s3-fs/tests/fuse_tests/metrics_test.rs

```diff
@@ -304,22 +308,21 @@ rusty_fork_test! {
         sleep(Duration::from_millis(100));
 
         // Verify cache population metrics
-        // Currently, disk_data_cache.read_duration_us is only recorded on cache hits, not misses.
         // But they are recorded for express.
         assert!(recorder.get("fuse.cache_hit", &[]).is_none(), "Cache hit metric should not exist after cache miss");
-        assert_metric_exists(&recorder, "disk_data_cache.write_duration_us", &[]);
-        assert_metric_exists(&recorder, "disk_data_cache.total_bytes", &[("type", "write")]);
-        assert_metric_exists(&recorder, "express_data_cache.write_duration_us", &[("type", "ok")]);
-        assert_metric_exists(&recorder, "express_data_cache.total_bytes", &[("type", "write")]);
-        assert_metric_exists(&recorder, "express_data_cache.read_duration_us", &[("type", "miss")]);
+        assert_metric_exists(&recorder, CACHE_PUT_LATENCY, &[(ATTR_CACHE, CACHE_DISK)]);
+        assert_metric_exists(&recorder, CACHE_PUT_IO_SIZE, &[(ATTR_CACHE, CACHE_DISK)]);
+        assert_metric_exists(&recorder, CACHE_GET_LATENCY, &[(ATTR_CACHE, CACHE_DISK)]);
+        assert_metric_exists(&recorder, CACHE_PUT_LATENCY, &[(ATTR_CACHE, CACHE_EXPRESS)]);
+        assert_metric_exists(&recorder, CACHE_PUT_IO_SIZE, &[(ATTR_CACHE, CACHE_EXPRESS)]);
+        assert_metric_exists(&recorder, CACHE_GET_LATENCY, &[(ATTR_CACHE, CACHE_EXPRESS)]);
 
         // Second read should hit disk cache after async write completes
         File::open(&path).unwrap().read_to_end(&mut Vec::new()).unwrap();
         sleep(Duration::from_millis(100));
 
         assert_metric_exists(&recorder, "fuse.cache_hit", &[]);
```

Comment:
acknowledging that we should look at `CACHE_GET_IO_SIZE` metric if we wanted to check, which cache is utilised (where "hits" come from)